javascript/javascript3/week3/githubrepos.js

```diff
@@ -0,0 +1,31 @@
+let promise1 = fetch(`https://api.github.com/search/repositories?q=user:rajani`)
+let promise2 = fetch(`https://api.github.com/search/repositories?q=user:sandhana`)
+let promise3 = fetch(`https://api.github.com/search/repositories?q=user:jyothinandyala`)
+Promise.all([promise1, promise2, promise3])
+    .then(files => {
+        files.forEach(file => {
+            renderPromises(file.json())
+        })
+    })
+    .catch(err => {
+        console.log(err)
+    })
+
+let renderPromises = (prom) => {
```

Comment:
`let` could be turned into a `const`. Function definitions are almost never changed.